src/Pos/WorkMode.hs

```diff
@@ -149,8 +108,8 @@ type RawRealMode ssc =
     SscHolder ssc (
     NtpSlotting (
     SlottingHolder (
-    ContextHolder ssc (
     DBHolder (
+    ContextHolder ssc (
```

Comment:
I implemented `ContextHolder` as a "catch-all" for `MonadReader`. Whenever we have `MonadReader r (ContextHolder ssc ...)`, this `r` should be in the context (there should be a `ContextPart` instance from the context to `r`).

When I changed `MonadDB` to `MonadReader`, it started to look for `NodeDBs` in the context. While we could put them there, for now we keep them in a separate layer.